admin/src/components/DeletedUserFormular.vue

```diff
@@ -4,15 +4,21 @@
       {{ $t('removeNotSelf') }}
     </div>
     <div v-else class="mt-5">
-      <b-form-checkbox switch size="lg" v-model="checked">
-        <div>{{ item.deletedAt ? $t('undelete_user') : $t('delete_user') }}</div>
-      </b-form-checkbox>
-
       <div class="mt-3 mb-5">
-        <b-button v-if="checked && item.deletedAt === null" variant="danger" @click="deleteUser">
+        <b-button
+          v-if="item.deletedAt === null"
```

Comment:
Not certain if necessary but what happen if deletedAt is undefined? Following suggestion catches deletedAt = null, deletedAt = undefined and also deletedAt = 0.
```suggestion
          v-if="!item.deletedAt"
```